doc/reviewing_PR_guide.md

```diff
@@ -0,0 +1,91 @@
+
+## 7. Failing CI tests aren't related
+
+We have many CI tests that run for each PR.  If you're going to give a PR a passing review, *you must have looked at these*.  I'm lazy too, I know it's tempting to ignore them, but please don't.
+
+# Indicators to call for a second opinion
+
+These aren't part of the review checklist, but they constitute observations that would justify getting more than one person with merge permissions to weigh in on the change.  **These don't mean your PR isn't getting merged**.  They're just things we've noticed are often associated with PRs that become problems later.
+
+- PR creator is resistant to changes suggested to PR, particularly from collaborators and up.
+- PRs that are part of a large, sweeping overhaul, without an issue posted or similar document discussing the design direction and plan for the overhaul.
+- Changes justified by what "players" need or want, without deeper rationale (contrary to popular belief, improving player experience *is* desirable, but our experience is that just 'asking people' will produce wildly different results depending on which people you're talking to, and changes justified this way rarely have much statistical rigour).
+- Changes that revert previously controversial adjustments with new math; this can be a hard one to know, but for example, if someone starts bumping up bow damage based on their research, the bar to merge is *very high*, as this is a topic that's been researched *to death* by now and it's highly unlikely - although not at all impossible - that a new contributor is aware of all the work done before.
+- Changes to any political hot-button topics.  These are frequent troll bait.  Anything that touches on pride flags, trans/gay rights, immigration, and real-life political figures should be watched closely.
+- Similarly, overly edgy content - especially stuff that relates to cannibalism, NPC servitude, etc should be monitored very closely.
+- Topical humour, memes, pop-culture references, etc. are often added with the best of intentions but their expiry date generally runs out before the PR has even finished merging.  They should be taken with a large grain of salt, even when going into something like Crazy Cataclysm.
+
+# The role of Discord and the Discord roles
+
+We don't expect all contributors to be active on Discord, but the [development Discord server](https://discord.gg/9qUg4Hyejv) is the fastest and most reliable way to find out who to talk to if your PR is having problems, or if you are a reviewer trying to call attention to a PR or ask about these problems.  Especially in the #development channel, there are very few questions on these topics that will get you in trouble.
```

Comment:
I'd maaaaaybe deduplicate this by referring to the README which has the devcord listed. For example:

```suggestion
We don't expect all contributors to be active on Discord, but the official Discord server (see [README.md](../README.md)) is the fastest and most reliable way to find out who to talk to if your PR is having problems, or if you are a reviewer trying to call attention to a PR or ask about these problems.  Especially in the #development channel, there are very few questions on these topics that will get you in trouble.
```

And I hate to play libre fanatic here, but not everyone has discord. As a FOSS project you will inevitably attract people who will contribute greatly but are averse to discord because of many reasons (like me!). There should be an alternative listed (e.g IRC or maybe even the discourse but that seems less appropriate) but: 1. I do not consider the IRC <-> discord bridge super stable 2. there are very few devs left on IRC.

This may be something to strongly consider for the future at least. Mentioning the IRC should not hurt but as of right now I'd put a disclaimer into the sentence that bridging may fail.

Reply:
I don't personally like the reliance either, but finding an alternate solution is outside the scope of this documentation. It's actually something we've been discussing though.